tic-tac-toe/src/redux/Login/reducer.js

```diff
@@ -1,28 +1,30 @@
-import { LOGGEDIN, NOAUTH, LOADING, OUT, ERROR } from '@constants/const';
-
 import { actions } from './actions';
 
 const initialState = {
-  loggedEstate: OUT
+  loginLoading: false,
+  loginSuccess: false,
+  errorAuthMessage: null
 };
 
 const reducer = (state = initialState, action) => {
```

Comment:
Excellent reducer! check that you are not using CHANGE action in this.. that's a clear clue that you don't really need it for this flow. What do you think?